solana/program_v2_cli/README.md

```diff
@@ -0,0 +1,247 @@
+oclif-hello-world
+=================
+
+oclif example Hello World CLI
```

Comment:
@DiosAkhilleus I mentioned it somewhere else, but lets pls get rid of this `hello world` type stuff

Reply:
Removed